engine/src/main/java/org/terasology/world/block/BlockRegion.java

```diff
@@ -413,34 +455,58 @@ public Vector3i getSize(Vector3i dest) {
     }
 
     /**
-     * The number of blocks on the X axis
-     *
-     * @return number of blocks in the X axis
+     * The number of blocks on the x axis.
      */
     public int sizeX() {
         return this.maxX - this.minX + 1;
     }
 
     /**
-     * The number of blocks on the Y axis
-     *
-     * @return number of blocks in the Y axis
+     * @see #sizeX()
+     */
+    public int getSizeX() {
+        return sizeX();
+    }
+
+    /**
+     * The number of blocks on the y axis.
```

Comment:
There is really no need to offer both. it's either getSizeX or sizeX().

Reply:
Yep, will remove `sizeX()` again as per comment on Discord: "only use `minX` (without `get` for coordinates and/or main attributes?)".